README.md

```diff
@@ -480,6 +481,24 @@ Resolver tests make use of mocks to simulate interaction with datasources. These
 
 To run the unit tests `npm run test`
 
+### Connect to ECS instance
```

Comment:
I decided to add this after working with Terry on how to connect to his ECS containers